sigopt_sklearn/search.py

```diff
@@ -331,20 +338,25 @@ def _fit(self, X, y, parameter_iterable=None):
                 for sidx, suggestion in enumerate(suggestions):
                     out_sidx = n_folds * sidx
                     scores = [o[0] for o in out[out_sidx:(out_sidx+n_folds)]]
-                    conn.experiments(self.experiment.id).observations().create(
-                        suggestion=suggestion.id,
-                        value=numpy.mean(scores),
-                        value_stddev=numpy.std(scores)
-                    )
+                    self.sigopt_connection \
+                        .experiments(self.experiment.id) \
+                        .observations() \
+                        .create(suggestion=suggestion.id,
+                                value=numpy.mean(scores),
+                                value_stddev=numpy.std(scores))
```

Comment:
The first form is what we typically use. There should be a `lint` executable at the top-level of this project